platform/wisdomctl/wisdomctl.py

```diff
@@ -0,0 +1,31 @@
+from ocifs import OCIFileSystem
+
+import click
+
+@click.group()
+def cli(): pass
+
+REMOTE = click.argument("remote")
+
+MB = 1024 ** 2
+def slowCopy(src, dest):
```

Comment:
I know its called slowCopy, but this looksw... terribele ;-)

We could merge this as is, but there should be a better way?

Reply:
I couldn't find a better way in the `ocifs` documentation. They do offer an optimized bucket-to-bucket copy. I wrote `slowCopy` to avoid the bottleneck you mentioned regarding memory allocation; the script should stay at low memory usage even when uploading/downloading GiB of data.